schema/organization/Person.md

```diff
@@ -0,0 +1,130 @@
+# Person
+
+The `Person` type allows you to provide details about a person such as their given and family names, any honourifix prefix or suffix, and contact details such as an email address. This type of often used to describe the `authors` of an [`Article`](/Article), or other [`CreativeWork`](/CreativeWork).
+
+## Examples
+
+These examples, based on Marie Curie, illustrate alternative ways for specifying a `Person`. [Marie Curie](https://en.wikipedia.org/wiki/Marie_Curie) was the first woman to win a Nobel Prize, the first person and only woman to win twice, and the only person to win a Nobel Prize in two different sciences.
+
+<!--
+These examples will eventually be wrapped in React components
+to illustrate how the input is converted into Stencila JSON
+See https://github.com/stencila/schema/issues/45
+-->
+
+Dr Curie can be represented in canonical Stencila JSON by:
+
+```json
+{
+  "type": "Person",
+  "honorificPrefix": "Dr",
+  "givenNames": ["Marie", "Skłodowska"],
+  "familyNames": ["Curie"],
+  "honorificSuffix": "PhD"
+}
+```
+
+YAML provides a more readable format for providing details about a person in places like Markdown front-matter. In the following example, we take advantage of [property aliases](/docs/property-aliases) to use the shorter `prefix` and `suffix` property names, and the US convention of `fistNames` and `lastName` (instead of `givenNames` and `familyNames`). We also use [property parsing](/docs/property-parsing) to be be able to write `fistNames` as a space separated values.
```

Comment:
Still mildly torn about us using YAML in documenting JSON schemas. I support the readability, but think it adds some mental overhead for readers trying to convert it JSON and understand the schema. Also prevents straight copy-pasting to experiment with.
I'm ok with keeping it as is for now, and then soliciting feedback/seeing what we think down the road.

minor typo: 
```suggestion
YAML provides a more readable format for providing details about a person in places like Markdown front-matter. In the following example, we take advantage of [property aliases](/docs/property-aliases) to use the shorter `prefix` and `suffix` property names, and the US convention of `firstNames` and `lastName` (instead of `givenNames` and `familyNames`). We also use [property parsing](/docs/property-parsing) to be be able to write `firstNames` as a space separated values.
```